internal/kafka/internal/services/kafka.go

```diff
@@ -1037,9 +1039,10 @@ func buildManagedKafkaCR(kafkaRequest *dbapi.KafkaRequest, kafkaConfig *config.K
 		"bf2.org/kafkaInstanceProfileQuotaConsumed": strconv.Itoa(k.QuotaConsumed),
 		"bf2.org/kafkaInstanceProfileType":          kafkaRequest.InstanceType,
 	}
-	if strings.HasPrefix(kafkaRequest.Status, "suspend") {
-		labels["bf2.org/suspended"] = fmt.Sprintf("%t", true)
-	}
+
+	suspendedStatuses := []string{constants.KafkaRequestStatusSuspending.String(), constants.KafkaRequestStatusSuspended.String()}
+	labels["bf2.org/suspended"] = fmt.Sprintf("%t", arrays.Contains(suspendedStatuses, kafkaRequest.Status))
```

Comment:
Thanks @pawelpaszki 

Do you think this code https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/c030410217731ebb6c3e6dce12ddcd83c1766beb/internal/kafka/internal/services/kafka.go#L1038-L1044

could be written as
```go
suspendedStatuses := []string{constants.KafkaRequestStatusSuspending.String(), constants.KafkaRequestStatusSuspended.String()}

labels := map[string]string{
		"bf2.org/kafkaInstanceProfileQuotaConsumed": strconv.Itoa(k.QuotaConsumed),
		"bf2.org/kafkaInstanceProfileType":          kafkaRequest.InstanceType,
                "bf2.org/suspended":  fmt.Sprintf("%t", arrays.Contains(suspendedStatuses, kafkaRequest.Status))
	}
```